www/ssc/Portfile

```
@@ -0,0 +1,40 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem                  1.0
+PortGroup                   github 1.0
+PortGroup                   cmake 1.1
+revision                    0
+
+github.setup                devongarde ssc 0.1.56 v
+github.tarball_from         releases
```

Please add the `revision 0` line. Even though that is the default, it makes it easier for humans or automated scripts to see where a revision should be incremented, when such a need arises.

Since `github.tarball_from` takes the place of `master_sites`, I recommend moving it below the `homepage`, where the `master_sites` line typically lives.

```suggestion
github.setup                devongarde ssc 0.1.56 v
revision                    0

```